package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "lint-staged": "^12.4.1"
+  },
+  "dependencies": {
+    "socket.io-client": "^4.5.1"
+  }
+}
```

Comment:
seems like you might've run `npm install` from the root folder or something
we shouldn't have a `package.json` in the root. did you mean to put these in the client package.json maybe?

Reply:
also delete the package-lock in the root
happy to help with this if needed, just lmk